.travis.yml

```diff
@@ -51,68 +51,24 @@ install:
         export PATH="/usr/local/opt/ccache/libexec:$PATH"
         && brew link --overwrite gcc;
     fi
-  - export CURR_DIR=$PWD
-  - cd $HOME/install
 # OCCA
-  - if [[ -d occa ]]; then
-        git -C occa pull --allow-unrelated-histories;
-        if [[ $? -eq 0 ]]; then
-          git -C occa reset --hard $OCCA_HEAD;
-        else
-          rm -rf occa
-          && git clone https://github.com/libocca/occa.git
-          && git -C occa reset --hard $OCCA_HEAD;
-        fi
-    else
-        git clone https://github.com/libocca/occa.git
+  - git clone https://github.com/libocca/occa.git
```

Comment:
All of the `git` logic was still running into problems, so I stopped caching anything but PETSc. Travis runs in about the same time.